### ISSUE TITLE
Model is overfitting on SHG skin images

### ISSUE THREAD

**Issue by siemdejong** [siemdejong/skinstression#7]
The model is overfitting on THG skin images.

![image](https://user-images.githubusercontent.com/28396796/193233030-c60ff3e6-7111-43bf-8dbe-ecc6248da330.png)

Try to resolve with
- [x] kfold does the shuffling part. Use Subset instead of SubsetRanomSampler?
- [x] more training data (getting data from z-stack e.g.)
- [x] more dropout
- [x] more data augmentation (color jitter? brightness?)
- [x] double check model layers
- [x] check right batch sizes
- [ ] properly normalize data?
- [x] do a good hyperparameter search
- [ ] pick only a subset (say the middle part) of the z-stack
